### PR TITLE
Rib 90 fix image upload

### PR DESCRIPTION
**Pull request by BenediktVitek**
- Getting images now using API endpoint, calling google drive directly
- Place image using thymeleaf to load image
- Place thumbnails now loaded when interest page is loaded so there is no delay when using searchbar

### CODE REVIEW

File: src/main/java/it/rate/webapp/services/GoogleImageService.java [hunk 1]
```diff
@@ -52,6 +53,18 @@ public String savePlaceImage(MultipartFile image, Long placeId) throws IOExcepti
     }
   }
 
+  public byte[] getImageById(String imageId) throws IOException {
+
+    try {
```
Comment (marthubner) [inline src/main/java/it/rate/webapp/services/GoogleImageService.java:58]: Does try inside of another try do something special?

File: src/main/java/it/rate/webapp/services/GoogleImageService.java [hunk 2]
```diff
@@ -52,6 +53,18 @@ public String savePlaceImage(MultipartFile image, Long placeId) throws IOExcepti
...
 
+  public byte[] getImageById(String imageId) throws IOException {
+
+    try {
+      try (InputStream imageStream =
+          driveService.files().get(imageId).executeMediaAsInputStream()) {
+        return imageStream.readAllBytes();
+      }
+    } catch (IOException e) {
+      throw new IOException("Could not retrieve image from server");
```
Comment (marthubner) [inline src/main/java/it/rate/webapp/services/GoogleImageService.java:64]: Maybe not necessary to catch IOException to throw it anyway.

Comment (dondowoldo) [inline src/main/java/it/rate/webapp/services/GoogleImageService.java:64]: We can throw api exception here instead when this is merged into develop

**Review by dondowoldo — APPROVED** [review]

**Review by marthubner — APPROVED** [review]

File: src/main/java/it/rate/webapp/services/GoogleImageService.java (hunk 1 above)
Comment (BenediktVitek) [reply src/main/java/it/rate/webapp/services/GoogleImageService.java]: whole method changed to implement new exceptions

File: src/main/java/it/rate/webapp/services/GoogleImageService.java (hunk 2 above)
Comment (BenediktVitek) [reply src/main/java/it/rate/webapp/services/GoogleImageService.java]: whole method changed to implement new exceptions

Comment (BenediktVitek) [reply src/main/java/it/rate/webapp/services/GoogleImageService.java]: whole method changed to implement new exceptions

**Pull request merged**